server/mailet/integration-testing/src/test/java/org/apache/james/mailets/BounceIntegrationTest.java

```diff
@@ -86,6 +92,46 @@ public void dsnBounceMailetShouldDeliverBounce() throws Exception {
             .awaitMessage(awaitAtMostOneMinute);
     }
 
+    @Test
+    public void dsnBounceMailetShouldDeliverBounceToTheMailFromAddress() throws Exception {
+        setup(DSNBounce.class);
+
+        messageSender.connect(LOCALHOST_IP, jamesServer.getProbe(SmtpGuiceProbe.class).getSmtpPort())
+                .sendMessageWithHeaders(SENDER, RECIPIENT,
+                        "From: " + OTHER + "\r\n" +
+                                "To: " + RECIPIENT + "\r\n" +
+                                "Subject: " + "Hello\r\n" +
+                                "\r\n" +
+                                "Please bounce me to the return address\r\n"
+                );
+
+        testIMAPClient.connect(LOCALHOST_IP, jamesServer.getProbe(ImapGuiceProbe.class).getImapPort())
+                .login(SENDER, SENDER_PASSWORD)
+                .select(TestIMAPClient.INBOX)
+                .awaitMessage(awaitAtMostOneMinute);
```

Comment:
I feel like in your last PRs I can see more and more indents with 8 spaces in your java code, while I believe the standard in James is 4? Maybe you need to change that config in your IDE?

Reply:
I think i have updated it, but i will check it out :)